infrastructure/resource_group_level/main.bicep

```diff
@@ -318,7 +318,7 @@ var parametersMap = {
       addressPrefixes: ['172.16.40.0/22', '172.16.60.0/23']
       applicationGatewayCidr: '172.16.41.0/24'
       appServicePlanEgressPrefixCidr: '172.16.42.0/28'
-      aciCidr: '172.16.42.16/29'
+      acaCidr: '172.16.42.32/27'
```

Comment:
Azure Container environments need a /27 subnet size (32 IPs). (You need to start a /27 on a multiple of 32.)